src/Library/demos/Link Button/code.rs

```diff
@@ -0,0 +1,15 @@
+use crate::workbench;
+use gtk::prelude::*;
+
+pub fn main() {
+    let linkbutton: gtk::LinkButton = workbench::builder().object("linkbutton").unwrap();
+    linkbutton.connect_clicked(move |_| {
+        println!("Link has been visited");
+    });
+
+    linkbutton.connect_activate_link(|button| {
+        println!("About to activate {}", button.uri());
+
```

Comment:
Please add the same comment of the javascript version: https://github.com/sonnyp/Workbench/blob/4169b5d9b27b6bd77bbc10bded7ff77f7fdadb1a/src/Library/demos/Link%20Button/main.js#L10

Reply:
Ive added the comments